pandas/tests/plotting/test_backend.py

```diff
@@ -80,7 +80,7 @@ def test_setting_backend_without_plot_raises(monkeypatch):
     assert pandas.options.plotting.backend == "matplotlib"
 
 
-@td.skip_if_mpl
+@td.skip_if_installed("matplotlib")
```

Comment:
is there any perf impact of evaluating these once vs in a bunch of places?

Reply:
Very slight. e.g. when running `test_eval.py` with `skip_if_no_ne` replaced locally I'm getting a runtime of 26.86s on this PR vs 26.69s on main. IMO I think this timing difference is OK given the cleanup